sklearn/preprocessing/data.py

```diff
@@ -633,15 +654,18 @@ def partial_fit(self, X, y=None):
             if self.with_std:
                 # First pass
                 if not hasattr(self, 'n_samples_seen_'):
-                    self.mean_, self.var_ = mean_variance_axis(X, axis=0)
+                    self.mean_, self.var_ = \
+                        mean_variance_axis(X, axis=0,
+                                           accepting_nan=accepting_nan)
```

Comment:
I would rename `accepting_nan` by `ignore_nan` for those cython function.
In fact, they internally ignore nan.